examples/Mamico-couette/ls1configNoCP.xml

```diff
@@ -0,0 +1,126 @@
+      <phasespacepoint>
+				<generator name="MultiObjectGenerator">
+					<objectgenerator>
+						<filler type="GridFiller">
+							<lattice system="cubic" centering="face">
+								<vec id="a"> <x>1.0714</x> <y>0</y> <z>0</z> </vec>
+								<vec id="b"> <x>0</x> <y>1.0714</y> <z>0</z> </vec>
+								<vec id="c"> <x>0</x> <y>0</y> <z>1.0714</z> </vec>
+							</lattice>
+							<basis>
+								<site>
+									<componentid>0</componentid>
+									<coordinate> <x>0.5</x> <y>0.5</y> <z>0.5</z> </coordinate>
+								</site>
+							</basis>
+							<latticeOccupancy>1</latticeOccupancy>
+							<!-- <density>0.73021</density> exact target (was 0,76)-->
+							<density>0.81</density> <!--while visualizing particles set size of particle to 6th root of 2 (1.2246) times sigma or sigma-->
+						</filler>
+						<object type="Cuboid">
+							<lower> <x>0</x> <y>0</y> <z>0</z> </lower>
+							<upper> <x>30</x> <y>30</y> <z>30</z> </upper>
+						</object>
+	<!--                <velocityAssigner type="MaxwellVelocityDistribution"></velocityAssigner>-->
+						<velocityAssigner type="EqualVelocityDistribution"></velocityAssigner>
+					</objectgenerator>
+				</generator>
+			</phasespacepoint>
+    </ensemble>
+    <algorithm>
+      <parallelisation type="DomainDecomposition"/>
+      <datastructure type="AutoPas">
+        <!--<cellsInCutoffRadius>1</cellsInCutoffRadius>-->
+      </datastructure>
+      <cutoffs type="CenterOfMass">
+        <radiusLJ unit="reduced">2.2</radiusLJ>
+      </cutoffs>
+      <electrostatic type="ReactionField">
+        <epsilon>1.0e+10</epsilon>
+      </electrostatic>
+
+   <!--   <thermostats>
+          <thermostat type="TemperatureControl">
+              <control>
+                  <start>0</start>
+                  <frequency>1</frequency>
+                  <stop>20000</stop>
+              </control>
+              <regions>
+                  <region>
+                      <coords>
+                          <lcx>0.0</lcx> <lcy>0.0</lcy> <lcz>0.0</lcz>
+                          <ucx>30</ucx> <ucy>30</ucy> <ucz>30</ucz>
+                      </coords>
+                      <target>
+                          <temperature>1.1</temperature>
+                          <component>0</component>
+                      </target>
+                      <settings>
+                          <numslabs>1</numslabs>
+                          <exponent>0.4</exponent>
+                          <directions>xyz</directions>
+                      </settings>
+                      <fileprefix>beta_log</fileprefix>
+                      <writefreq>100000000</writefreq>
+                  </region>
+              </regions>
+          </thermostat>
+      </thermostats>
+-->
+    </algorithm>
+    <output>
+      <outputplugin name="CheckpointWriter">
+        <type>binary</type>
+        <writefrequency>10000</writefrequency>
+        <outputprefix>cp_binary</outputprefix>
+      </outputplugin>
```

Comment:
my guess was "NoCP" in the filename meant no checkpoint. Apparently I was wrong :D
What is the purpose of the second config and do we actually need it?

Reply:
maybe add a readme to the directory describing the scenarios